packages/appsync-modelgen-plugin/src/visitors/appsync-visitor.ts

```diff
@@ -391,6 +391,8 @@ export class AppSyncModelVisitor<
             name: field.name,
             directives: fieldDirectives,
             type: field.type,
+            isNullable: field.isNullable,
+            isList: field.isList,
```

Comment:
I think there is one scenario missing for nullability for list. Suppose in the two schema the only difference is field `[String]` and `[String]!`. And this will have the same computed version. I suggest to add an additional line as following and related unit tests.
```suggestion
            isList: field.isList,
            isListNullable: field.isListNullable
```

Reply:
FYI `isListNullable` is an attribute for field after loading https://github.com/aws-amplify/amplify-codegen/blob/master/packages/appsync-modelgen-plugin/src/utils/get-type-info.ts

Reply:
Nice catch! I just added this with a corresponding test 👍